gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 518
+cordaApiRevision = 519
```

Comment:
Does it need the version bump?

Reply:
As it says in description "Potentially, this should be a non-breaking change.", but I want to test it first in `corda-runtime-os`.